sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/ActorOperations.java

```diff
@@ -158,7 +158,9 @@
   USER_TENANT_MIGRATE("userTenantMigrate"),
   GET_PARTICIPANTS("getParticipants"),
   FREEUP_USER_IDENTITY("freeUpUserIdentity"),
-  RESET_PASSWORD("resetPassword");
+  RESET_PASSWORD("resetPassword"),
+  MERGE_USER("mergeUser"),
+  MERGE_USERS_TO_ELASTIC("mergerUserSToElastic");
```

Comment:
MERGE_USERS_TO_ELASTIC to MERGE_USER_TO_ELASTIC and mergerUserSToElastic to mergerUserToElastic

Reply:
done